functional-tests/build.gradle

```diff
@@ -12,14 +12,18 @@ ext {
     drivers = ["firefox", "firefoxHeadless", "chrome", "chromeHeadless","ie","edge","safari"]
 
     ext {
-        groovyVersion = '2.4.12'
-        gebVersion = '2.0'
-        seleniumVersion = '3.6.0'
-        chromeDriverVersion = '2.36'
-        geckoDriverVersion = '0.18.0'
-        ieDriverVersion = '3.6'
-        edgeDriverVersion = seleniumVersion
-        safariDriverVersion = seleniumVersion
+        groovyVersion = '2.4.15'
+        gebVersion = '2.2'
+        seleniumVersion = '3.14.0'
+        chromeDriverVersion = '2.41'
+        geckoDriverVersion = '0.21.0'
+        ieDriverVersion = '3.14.0'
+        edgeDriverVersion = '3.14.0'
+        safariDriverVersion = '3.14.0'
+
+        spockCoreVersion = '1.1-groovy-2.4'
+        spockReportsVersion = '1.6.0'
+        slf4jApiVersion = '1.7.25'
```

Comment:
Just version updates, and pulling a few versions into variables rather than be inline below.